codersdk/deployment.go

```diff
@@ -199,7 +199,7 @@ func ParseSSHConfigOption(opt string) (key string, value string, err error) {
 		return r == ' ' || r == '='
 	})
 	if idx == -1 {
-		return "", "", fmt.Errorf("invalid config-ssh option %q", opt)
```

Comment:
Idk why this was replaced. Maybe a lint stuff?

Reply:
Yep, this is our linter. You should use `xerrors.Errorf` instead of `xerrors.New(fmt.Sprintf...`